src/cdk-experimental/menu/menu-item-trigger.spec.ts

```diff
@@ -293,6 +301,35 @@ describe('MenuItemTrigger', () => {
         .toEqual(Math.floor(nativeMenus[1].getBoundingClientRect().top));
     });
   });
+
+  describe('with shared triggered menu', () => {
+    /**
+     * Return a function which builds the given component and renders it.
+     * @param componentClass the component to create
+     */
+    function createComponent<T>(componentClass: Type<T>) {
+      return function () {
+        TestBed.configureTestingModule({
+          imports: [CdkMenuModule],
+          declarations: [componentClass],
+        }).compileComponents();
+
+        TestBed.createComponent(componentClass).detectChanges();
+      };
+    }
+
+    it('should throw an error if two triggers in different menubars open the same menu', () => {
+      expect(createComponent(TriggersWithSameMenuDifferentMenuBars)).toThrowError();
+    });
+
+    it('should throw an error if two triggers in the same menubar open the same menu', () => {
+      expect(createComponent(TriggersWithSameMenuSameMenuBar)).toThrowError();
+    });
+
+    it('should throw an error if a trigger in a submenu references its parent menu', () => {
+      expect(createComponent(TriggerOpensItsMenu)).toThrowError();
```

Comment:
Should we be testing for the specific error that we're expecting here? E.g. `.toThrowError(/CdkMenuPanel is already referenced by different CdkMenuTrigger/)`.